### ISSUE TITLE
OpenJ9 Crashes on Start Immediately

### ISSUE THREAD

**Issue by shanoaice** [CleanroomMC/Cleanroom#49]
Java Version: Java 21 JDK, IBM Semeru OpenJ9 JVM
Cleanroom Version: 15.24.0.3028
LWJGL Version: 3.3.4-27-CLEANROOM

The game crashes immediately when I started it with Prism Launcher. Log is here:
```
WARN StatusConsoleListener The use of package scanning to locate plugins is deprecated and will be removed in a future release
WARN StatusConsoleListener The use of package scanning to locate plugins is deprecated and will be removed in a future release
WARN StatusConsoleListener The use of package scanning to locate plugins is deprecated and will be removed in a future release
WARN StatusConsoleListener The use of package scanning to locate plugins is deprecated and will be removed in a future release
[ImagineBreaker] Opening modules natively...
[ImagineBreaker] Removing field reflection filters natively...
[ImagineBreaker] Removing method reflection filters natively...
Unhandled exception
Type=Segmentation error vmState=0x00040000
Windows_ExceptionCode=c0000005 J9Generic_Signal=00000004 ExceptionAddress=00007FFD7BC6660D ContextFlags=0010005f
Handler1=00007FFD7BC5B5D0 Handler2=00007FFDB4A5AC00 InaccessibleReadAddress=0000000000000010
RDI=0000000000021F00 RSI=0000000000000000 RAX=00007FFD7BD75270 RBX=00000295E35CDD60
RCX=0000000000021F00 RDX=0000000000140260 R8=0000000000000000 R9=0000000000000000
R10=0000000000989680 R11=000000E8F567E950 R12=0000000000140260 R13=0000000000000000
R14=000000E8F567EF70 R15=0000000000000000
RIP=00007FFD7BC6660D RSP=000000E8F567EAC0 RBP=000000E8F567EBD0 EFLAGS=0000000000010246
FS=0053 ES=002B DS=002B
XMM0 43e0000000000000 (f: 0.000000, d: 9.223372e+18)
XMM1 3fe3333333333333 (f: 858993472.000000, d: 6.000000e-01)
XMM2 4024000000000000 (f: 0.000000, d: 1.000000e+01)
XMM3 0000000000140260 (f: 1311328.000000, d: 6.478821e-318)
XMM4 0000000000000000 (f: 0.000000, d: 0.000000e+00)
XMM5 61657242656e6967 (f: 1701734784.000000, d: 1.507581e+161)
XMM6 0000000000000000 (f: 0.000000, d: 0.000000e+00)
XMM7 0000000000000000 (f: 0.000000, d: 0.000000e+00)
XMM8 0000000000000000 (f: 0.000000, d: 0.000000e+00)
XMM9 0000000000000000 (f: 0.000000, d: 0.000000e+00)
XMM10 0000000000000000 (f: 0.000000, d: 0.000000e+00)
XMM11 0000000000000000 (f: 0.000000, d: 0.000000e+00)
XMM12 0000000000000000 (f: 0.000000, d: 0.000000e+00)
XMM13 0000000000000000 (f: 0.000000, d: 0.000000e+00)
XMM14 0000000000000000 (f: 0.000000, d: 0.000000e+00)
XMM15 0000000000000000 (f: 0.000000, d: 0.000000e+00)
Module=D:\JavaRT\Semeru\jdk-21.0.2+13\bin\default\j9vm29.dll
Module_base_address=00007FFD7BB90000 Offset_in_DLL=00000000000d660d
Target=2_90_20240116_95 (Windows 11 10.0 build 23620)
CPU=amd64 (16 logical CPUs) (0x3d5f31000 RAM)
----------- Stack Backtrace -----------
(0x00007FFD7BC6660D [j9vm29+0xd660d])
Java_zone_rong_imaginebreaker_NativeImagineBreaker_clearReflectionData+0x25ea (0x00007FFDBA8D3B7A [imaginebreaker+0x3b7a])
Java_zone_rong_imaginebreaker_NativeImagineBreaker_clearReflectionData+0x51 (0x00007FFDBA8D15E1 [imaginebreaker+0x15e1])
ffi_call_go+0x3c1 (0x00007FFD7BD2CCA1 [j9vm29+0x19cca1])
ffi_call_go+0x213 (0x00007FFD7BD2CAF3 [j9vm29+0x19caf3])
ffi_call_go+0x22 (0x00007FFD7BD2C902 [j9vm29+0x19c902])
(0x00007FFD7BBAA6C6 [j9vm29+0x1a6c6])
(0x0000000000000001)
(0x000000E8F567F138)
ffi_call_go+0x1e958 (0x00007FFD7BD4B238 [j9vm29+0x1bb238])
(0x000000E8F567F138)
(0x000000E8F567ED30)
---------------------------------------
JVMDUMP039I 正在处理转储事件“gpf”，详细信息“”，位于 2024/02/08 18:10:24 - 请稍候。
JVMDUMP032I JVM 使用“MojangTricksIntelDriversForPerformance_javaw.exe_minecraft.exe.heapdump\core.20240208.181024.5260.0001.dmp”来请求 System 转储以响应事件
JVMDUMP010I System 转储已写入 MojangTricksIntelDriversForPerformance_javaw.exe_minecraft.exe.heapdump\core.20240208.181024.5260.0001.dmp
JVMDUMP032I JVM 使用“MojangTricksIntelDriversForPerformance_javaw.exe_minecraft.exe.heapdump\javacore.20240208.181024.5260.0002.txt”来请求 Java 转储以响应事件
JVMDUMP010I Java 转储已写入 MojangTricksIntelDriversForPerformance_javaw.exe_minecraft.exe.heapdump\javacore.20240208.181024.5260.0002.txt
JVMDUMP032I JVM 使用“MojangTricksIntelDriversForPerformance_javaw.exe_minecraft.exe.heapdump\Snap.20240208.181024.5260.0003.trc”来请求 Snap 转储以响应事件
JVMDUMP010I Snap 转储已写入 MojangTricksIntelDriversForPerformance_javaw.exe_minecraft.exe.heapdump\Snap.20240208.181024.5260.0003.trc
JVMDUMP032I JVM 使用“MojangTricksIntelDriversForPerformance_javaw.exe_minecraft.exe.heapdump\jitdump.20240208.181024.5260.0004.dmp”来请求 JIT 转储以响应事件
JVMDUMP051I JIT 转储出现在 'main' 线程 0x0000000000021F00 中
JVMDUMP010I JIT 转储已写入 MojangTricksIntelDriversForPerformance_javaw.exe_minecraft.exe.heapdump\jitdump.20240208.181024.5260.0004.dmp
JVMDUMP013I 已处理转储事件“gpf”，详细信息：“”。
```

Java Parameters:
```
-Xaggressive -Xgcpolicy:balanced -Xgc:concurrentScavenge -Xnoaot
```

**Comment by Rongmario** [comment]
Fixed in ImagineBreaker's dev as of this commit: https://github.com/Rongmario/ImagineBreaker/commit/d2e999266c41cefd220960bb9d1a20f994f1a5fa. Will close upon merge.

**Comment by Rongmario** [comment]
Fixed after ImagineBreaker 2.0 update.
